getnacelle/nuxt-reference-store
# LD 1104 Homepage

### WHY are these changes introduced?

Ticket [[LD-1104](https://nacelle.atlassian.net/browse/LD-1104)] 

### WHAT is this pull request doing?

- Adds homepage components: `Hero`, `FeaturedProducts`, and `SideBySideFull`
- Adds dynamic content from Sanity

### How to Test

1. Checkout branch `LD-1104_homepage`
2. `npm run dev` to start Nuxt
3. Open the site to the homepage to see sections filled with content from Sanity. Checkout https://github.com/getnacelle/nacelle-reference-store-sanity-studio/pull/3 to make changes to this content.
